packages/core/src/App/Containers/RealAccountSignup/currency-selector.jsx

```diff
@@ -207,10 +211,11 @@ class CurrencySelector extends React.Component {
                                 </Div100vhContainer>
                                 <FormSubmitButton
                                     is_disabled={isSubmitting || !values.currency}
-                                    is_center={!has_currency || isMobile()}
-                                    label={
-                                        !has_real_account && isMobile() ? localize('Next') : localize('Set currency')
-                                    }
+                                    is_center={!has_currency}
+                                    has_cancel={true}
+                                    onCancel={this.props.onCancel}
+                                    cancel_label={localize('Cancel')}
+                                    label={localize('Next')}
```

Comment:
Can you clarify`isMobile()` check is no longer required? Previously this was due to the differences in design for mobile and desktop in `CurrencySelector`.

Reply:
Design for desktop is updated.